version

```diff
@@ -1 +1 @@
-1.51
+2.0
```

Comment:
If you're up to the task, this file most likely doesn't need to exist.

My memory of the capabilities of Lua pattern matching have deteriorated significantly, but if my memory serves right , you can most likely fetch the raw `fxmanifest.lua` from GitHub and parse for the version field in the manifest file.